File: Java/MultiplosEnIntervalo.java
import java.io.*;

public class MultiplosEnIntervalo {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int a = Integer.parseInt(br.readLine());
        int b = Integer.parseInt(br.readLine());

        for (int i = a; i <= b; i++) {
            if (i % a == 0) {
                System.out.print(i + " ");
            }
        }
    }
}

// https://omegaup.com/arena/problem/MDUI-UAMA/#problems

File: Python/MultiplosEnIntervalo.py
def main():
    a = int(input())
    b = int(input())

    for i in range(a, b + 1):
        if i % a == 0:
            print(i, end=" ")   

main()

# https://omegaup.com/arena/problem/MDUI-UAMA/#problems